packages/frontend/src/components/accounts/CouldNotFindAccountModalWrapper.js

```diff
@@ -0,0 +1,44 @@
+import { KeyPair } from 'near-api-js';
+import { parseSeedPhrase } from 'near-seed-phrase';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+import {
+    redirectTo,
+    refreshAccount
+} from '../../redux/actions/account';
+import { showCustomAlert, clearGlobalAlert } from '../../redux/actions/status';
+import { getImplicitAccountIdFromSeedPhrase, getKeyPairFromSeedPhrase } from '../../utils/parseSeedPhrase';
+import { wallet } from '../../utils/wallet';
+import CouldNotFindAccountModal from './CouldNotFindAccountModal';
+
+export default ({
```

Comment:
Can we please use named functions so we get better visibility in react dev tools and avoid `_default`?

![image](https://user-images.githubusercontent.com/11974624/167484984-c3eec86b-c4eb-424a-a1d4-49ff4350f124.png)